docs/localInstallation.md

```diff
@@ -19,170 +19,183 @@ layout: default
       - [Web service folder](#web-service-folder)
       - [Web application folder](#web-application-folder)
       - [Database and ontologies](#database-and-ontologies)
-+ [Installation](#phis-installation)
-    - [Mongodb Database](#mongodb-database)
-    - [Postgresql Database](#postgresql-database)
++ [Installation](#opensilex-installation)
+    - [MongoDB Database](#mongodb-database)
+    - [PostgreSQL Database](#postgresql-database)
     - [Rdf4j ontologies](#rdf4j-ontologies)
     - [Web service](#web-service)
     - [Web application](#web-application)
 + [Current errors](#current-errors)
-    - [Problems with netbeans](#problems-with-netbeans)
+    - [Problems with Netbeans](#problems-with-netbeans)
     - [Problems with Composer](#problems-with-composer)
-    - [Errors with postgresql](#errors-with-postgresql)
+    - [Errors with PostgreSQL](#errors-with-postgresql)
     - [Errors with the web service](#errors-with-the-web-service)
     - [Errors with the web application](#errors-with-the-web-application)
-    - [Other problems concerned webapp and web service](#other-problems-concerned-webapp-and-web-service)
+    - [Other problems concerning the web application and the web service](#other-problems-concerned-webapp-and-web-service)
 
 ## Introduction  
-This document explains you how to deploy PHIS on your personnal computer.  
-In this document commands lines are for **Ubuntu 16.04**, but for the majority of them, they are compatible with all Debian distribution which have the package manager Aptitude.
+
+This document explains you how to deploy OpenSILEX on your personnal computer.  
+In this document, commands are for **Ubuntu 16.04**. However, the majority of these commands are compatible with all Debian distributions which have the package manager Aptitude.
 
 ## Prerequisite
 
 ### Software
 
-#### Mongodb + robo3t
+#### MongoDB + Robo 3T
```

Comment:
Possible conflict with 398c2f3e4c99094cde24e5d1ff2ed19be9914585 (it's true for the other modified headers also)